docs/Protected Services/index.md

```diff
@@ -0,0 +1,31 @@
+# Protected Services
+
+#### Related Evolution Proposals
+- [0317: SDL Protocol Security Specification](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0317-sdl-protocol-security-specification.md)
+
+## 1. Overview
+
+The authentication is done using TLS handshake. The TLS handshake process is defined by TLS and is not part of the SDL protocol.

```

Comment:
```suggestion
Secured services in SDL are established using a TLS handshake, the TLS handshake process is defined by TLS and is not part of the SDL protocol.
```

Reply:
@jacobkeeler fixed in https://github.com/smartdevicelink/sdl_overview_guides/pull/80/commits/6bb26e230744f696af3d4ca9befb0fb913cf08b8